oxide-auth/src/endpoint/mod.rs

```diff
@@ -121,7 +121,7 @@ pub enum ResponseStatus {
 /// not derive this until this has shown unlikely but strongly requested. Please open an issue if you
 /// think the pros or cons should be evaluated differently.
 #[derive(Debug)]
-enum InnerTemplate<'a> {
```

Comment:
I didn't catch that change in the first review. In general, I agree `From<_>` is a better constructor than the public methods duplicating all the variants on the type. However, the comment above explains the reasoning why it was chosen when the code was written, so please add `#[non_exhaustive]` and choose a better public name for the struct. Maybe `TemplateKind`, if you agree it is sufficiently clear (inspired by the split of  std::io::Error/std::io::ErrorKind)

Reply:
Not sure if I understand this correctly but I was able to revert the change from using `InnerTemplate` in favor of `Template` if that is okay.

Reply:
That's fine as well :+1: